include/RAJA/RAJA.hxx

```diff
@@ -152,6 +152,10 @@
 
 #endif  // defined(RAJA_ENABLE_NESTED)
 
+#if defined(RAJA_ENABLE_SCAN)
+
```

Comment:
Yup, correct. If you have a chance to look at the errors thrust throws that could be helpful.